disdrodb/l0/check_configs.py

```diff
@@ -6,6 +6,8 @@
 from typing import List, Union, Optional
 from pydantic import BaseModel, ValidationError, validator
 
+from disdrodb.l0.standards import available_sensor_name
```

Comment:
Side comment: `ruff `should not also do the sorting of the imports à la `isort`?

Reply:
In https://github.com/ltelab/disdrodb/blob/main/pyproject.toml#L75 add `select = ["F", "E", "W", "I001"]`.
E and F are already used by default, while adding `I001` let ruff do the work of isort.
Adding `W` make ruff automatically corrects for docstring errors: https://beta.ruff.rs/docs/rules/#pydocstyle-d
You can do that in a separate PR (one for correcting the imports) and one to correct the docstring (W).
Potentially of interest to clean the code out @regislon, we could also have a look at: 
- https://beta.ruff.rs/docs/rules/#flake8-pytest-style-pt
- https://beta.ruff.rs/docs/rules/#flake8-quotes-q
- https://beta.ruff.rs/docs/rules/#flake8-return-ret
- https://beta.ruff.rs/docs/rules/#flake8-simplify-sim
- https://beta.ruff.rs/docs/rules/#flake8-use-pathlib-pth
- https://beta.ruff.rs/docs/rules/#pyupgrade-up